src/webots/vrml/WbNode.cpp

```diff
@@ -26,12 +26,15 @@
 #include "WbMFString.hpp"
 #include "WbMFVector2.hpp"
 #include "WbMFVector3.hpp"
+#include "WbNetwork.hpp"
 #include "WbNodeFactory.hpp"
 #include "WbNodeModel.hpp"
 #include "WbNodeReader.hpp"
 #include "WbParser.hpp"
 #include "WbProject.hpp"
+#include "WbProtoManager.hpp"
 #include "WbProtoModel.hpp"
+#include "WbProtoTreeItem.hpp"
```

Comment:
```suggestion
```

Reply:
Since we moved `combinePath` to WbUrl, I don't think this import is necessary anymore